src/baseball/main.py

```
@@ -1,10 +1,49 @@
+    def randnumber():
+        return random.sample(range(1, 10), 3)  
+    while True:
+        arr = randnumber()
+
+        while True:
+            try:
+                num = int(input())  # 메시지 없이 입력받음
+                if num < 100 or num > 999:  # 3자리 숫자가 아닐 경우
+                    raise ValueError("3자리 숫자를 입력해야 합니다.")
+            except ValueError:
+                raise ValueError(("잘못된 입력입니다"))
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)